com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/util/TextureUtil.java

```diff
@@ -400,16 +401,28 @@ public static TextureGenerator.GenerateResult createMultiPageTexture(List<Buffer
 
     // Public api
     public static List<BufferedImage> loadImages(List<IResource> resources) throws IOException, CompileExceptionError {
-        List<BufferedImage> images = new ArrayList<BufferedImage>(resources.size());
-
-        for (IResource resource : resources) {
-            BufferedImage image = ImageIO.read(new ByteArrayInputStream(resource.getContent()));
-            if (image == null) {
-                throw new CompileExceptionError(resource, -1, "Unable to load image " + resource.getPath());
+        try {
+            return resources.parallelStream()
```

Comment:
There is no sense to read files one by one, parallel reading should always be faster or at least not slower in case of IO bottleneck

It's only for bob